docs/modules/transform-sync/how-it-works.md

```diff
@@ -46,10 +37,30 @@ component TransformInternal {
 There are a few reasons why you might want different `Transform` and `Position` components.
 
 1. **Separation of responsibilies.** The `Improbable.Position` component is the load balancer's representation of location and the `TransformInternal` component is the workers' representation of location. This allows you to abstract the concept of location from the load balancer. A gameplay programmer would only need to be aware of the `TransformInternal` component without having to know about the load balancer at all.
-2. **Save bandwidth.** The `Position` component represents location with 3 doubles. The `TransformInternal` component represents location as 3 floats, which is the Unity native representation. This means that, ignoring rotation, a position update is twice as large as a transform update. If the `TransformInternal` component has a high frequency update rate and the `Position` component has a low frequency update rate then you have a net bandwidth saving over updating just the `Position` component at a high frequency!
-3. **Atomicity.** The `Transform` component contains more than just the location. If the other fields were on a separate component, you lose the guarantee that all fields are updated atomically.<br/><br/>Two updates sent in the same frame from one worker are _not_ guaranteed to be received on the same frame in another.
+2. **Save bandwidth.** The `TransformInternal` component contains the compressed representations of location, rotation and velocity. By comparison, a `Position` uses 3 doubles representing the raw location of an entity. This means that if the `TransformInternal` component has a high frequency update rate and the `Position` component has a low frequency update rate, you have a net bandwidth saving compared to updating just the `Position` component at a high frequency!
+3. **Atomicity.** The `TransformInternal` component contains more than just the location. If the other fields were on a separate component, you lose the guarantee that all fields are updated atomically.Two updates sent in the same frame from one worker are _not_ guaranteed to be received in the same frame on another.
```

Comment:
```suggestion
3. **Atomicity.** The `TransformInternal` component contains more than just the location. If the other fields were on a separate component, you lose the guarantee that all fields are updated atomically. Two updates sent in the same frame from one worker are _not_ guaranteed to be received in the same frame on another.
```

Reply:
this was fixed locally and pushed in a newer commit